Userland/Libraries/LibGfx/Size.h

```diff
@@ -164,6 +164,12 @@ class Size {
 
     [[nodiscard]] String to_string() const;
 
+    template<typename U>
+    [[nodiscard]] Size<U> to_rounded() const
+    {
+        return Size<U>(roundf(width()), roundf(height()));
```

Comment:
Using `roundf` here without constraining `T` is a bit of an assumption; maybe you could go with `AK::round_to` and if required, then specialize if it provides any measurable advantages? (Or improve `AK::round_to` of course)

Reply:
I'll be honest, I only used `roundf` because that's what `.to_rounded()` did for `Gfx::Point` (without any other specialisations).  

I don't really need the floating point rounds (i.e. when the target type is a float), so I could maybe add something like: 
```c++
    template<Integral I>
    [[nodiscard]] Size<I> to_rounded() const
    {
        return Size<I>(round_to<I>(width()), round_to<I>(height()));
    }
```
and leave the other specialisations until they're needed.

Reply:
That seems fine!